packages/terra-notification-dialog/translations/sv.json

```
@@ -1,6 +1,6 @@
 {
-  "Terra.notification.dialog.hazard-low": "Information",
+  "Terra.notification.dialog.hazard-low":	"Information",
```

Should that just be a space?
```suggestion
  "Terra.notification.dialog.hazard-low": "Information",
```